src/javascript/JContent/EditFrame/Create.jsx

```diff
@@ -15,15 +15,13 @@ import {SavePropertiesMutation} from '~/ContentEditor/ContentEditor/updateNode/u
 import {triggerRefetchAll} from '~/JContent/JContent.refetches';
 
 const ButtonRenderer = getButtonRenderer({
-    showTooltip: true,
-    defaultButtonProps: {color: 'default'},
-    defaultTooltipProps: {placement: 'top', classes: {popper: styles.tooltipPopper}}}
-);
+    showTooltip: false,
```

Comment:
disabling tooltips here is intentional ?

Reply:
@gflores-jahia yes, as it looked like this:
![image](https://github.com/user-attachments/assets/88d87e2c-c844-4b32-8f31-5a7c0923f7f2)
And me and Felix agreed that this is not needed.